internal/csi-addons/rbd/replication.go

```diff
@@ -326,7 +326,23 @@ func (rs *ReplicationServer) DisableVolumeReplication(ctx context.Context,
 	case librbd.MirrorImageDisabling:
 		return nil, status.Errorf(codes.Aborted, "%s is in disabling state", volumeID)
 	case librbd.MirrorImageEnabled:
-		return corerbd.DisableVolumeReplication(rbdVol, mirroringInfo, force)
+		err = rbdVol.DisableVolumeReplication(mirroringInfo, force)
+		if err == nil {
+			return &replication.DisableVolumeReplicationResponse{}, nil
+		}
+		switch {
+		case strings.Contains(err.Error(), "failed to get local state"):
```

Comment:
Instead of checking for contents of the string, use `errors.Is()` as function to check if an error is of a particular type.

https://github.com/ceph/ceph-csi/blob/40888f01b66245b3b15042b62903ae742cae494a/internal/rbd/clone.go#L58-L88